README.md

```diff
@@ -764,6 +766,51 @@ It becomes then possible to retrieve past values :
 - ```<name[0]>``` uses the first value taken by the key
 - ```<name[1]>``` uses the second element taken by the key
 
+## JSON matchers
+
+The built-in JSON DSL makes it possible to express detailed assertions regarding the values of JSON fields.
+
+If a field is unstable you can also decide to ignore it completely but what if you do not care about the precise value but still want to make sure it has a certain property.
+
+JSON matchers are designed for this use case, they work more or less like placeholders in practice.
+
+```scala
+And assert body.ignoring("city", "realName", "publisher.location").is(
+  """
+  {
+    "name": "<<any-alphanum-string>>",
+    "hasSuperpowers": "<<any-boolean>>",
+    "publisher": {
+      "name": "<<any-alphanum-string>>",
+      "foundationYear": "<<any-positive-integer>>"
+    }
+  }
+  """
+)
+
+```
+
+You just need to replace the value of the field by one of the built-in JSON matchers.
+
+There is a caveat though, unlike placeholders, the JSON matchers must be surrounded by quotes in all circumstances even if they are replacing JSON numbers or boolean.
+
+Here are the available matchers:
+
+- ```<<is-present>>``` : checks if the field is defined
+- ```<<any-array>>``` : checks if the field is an Array
+- ```<<any-object>>``` : checks if the field is an Object
+- ```<<any-integer>>``` : checks if the field is an integer
+- ```<<<any-positive-integer>>>``` : checks if the field is a positive integer
+- ```<<any-negative-integer>>``` : checks if the field is a negative integer
+- ```<<any-uuid>>``` : checks if the field is a valid UUID
+- ```<<any-boolean>>``` : checks if the field is a boolean
+- ```<<any-alphanum-string>> : checks if the field is an alpha-numeric string
+- ```<<any-date>>``` : checks if the field is a 'yyyy-MM-dd' date
+- ```<<any-date-time>>``` : checks if the field is a 'yyyy-MM-dd HH:mm:ss.SSS' datetime
```

Comment:
I'd try to match any ISO 8601 date (with timezone), or at least UTC (with a `Z` in the end).